ui/plugins/failure_summary/controller.js

```diff
@@ -63,7 +63,21 @@ treeherder.controller('BugsPluginCtrl', [
                             );
                             suggestions.push(suggestion);
                         });
+                        var errors = [];
+                        if (suggestions.length === 0 && artifact_list.length > 1) {
+                            var artifact = artifact_list[1];
+                            angular.forEach(artifact.blob.step_data.steps, function(step) {
+                                if (step.result !== "success") {
+                                    errors.push({
+                                        "name": step.name,
+                                        "result": step.result,
+                                        "line": step.finished_linenumber
```

Comment:
Here you would call a function to get the log viewer url like this: ``"lvUrl": thUrl.getLogViewerUrl(job_id)+"#"+step.finished_linenumber``.  This is how we do it in the ``plugins/controller`` to get the regular logviewer URL.